Readme.md

```diff
@@ -46,6 +46,15 @@ Similarly, you can execute `npm run dotnet-postclean --if-present` via the `dotn
 ```
 
 
+## Using pnpm for package restore
+
+[pnpm](https://pnpm.io/) is a faster and more efficient package manager. If it's installed globally, then the module will use that instead of *npm* to restore packages.
```

Comment:
```suggestion
[Pnpm](https://pnpm.io/) is a faster and more efficient package manager. If it's installed globally, then the module will use that instead of *npm* to restore packages.
```